ios/KaMPStarteriOS/ViewController.swift

```diff
@@ -14,16 +14,19 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
     @IBOutlet weak var breedTableView: UITableView!
     var data:[Breed] = []
     
-    private var model: BreedModel?
     let log = KoinIOS().get(objCClass: Kermit.self, parameter: "ViewController") as! Kermit
+    
+    private var adapter: NativeCoroutineAdapter?
```

Comment:
Would doing something like `lazy var adapter = NativeCoroutineAdapter(...)` let us avoid making this nullable? Or does that not work with the self references in the update lambdas or something?

Reply:
Made Lazy